### ISSUE TITLE
Subtitles download 

### ISSUE THREAD

**Issue by hvroun1** [jmir1/aniyomi-mpv-beta#39]
Since now we can stream with subtitles, is there any way we can also download with subtitles?

**Comment by ugwapings** [comment]
i also want to know ;)

**Comment by sinnghabhay80** [comment]
It should be happening automatically now as it has been implemented already.

**Comment by hvroun1** [comment]
@sinnghabhay80, do you mean that the downloaded episodes to watch locally offline, can now have subtitles?

**Comment by sinnghabhay80** [comment]
Yup it has been implemented and I also used it in zoro.to extension and it is working.

**Comment by hvroun1** [comment]
oh, i see. Maybe the issue is with the extension im using. I tried with superstream but unfortunately it seems its not working yet.

**Comment by sinnghabhay80** [comment]
I didn't download a complete file but with a partial download it seems that the subtitles are there in superstream.![Screenshot_2023-01-17-14-34-50-73_4aed3257f278fcf7bfa3abd644e23333.jpg](https://user-images.githubusercontent.com/55084281/212857231-cf96a0b7-51cd-4926-b524-57545f595198.jpg)

**Comment by hvroun1** [comment]
it shows that only while streaming, but once downloaded all options disappear.   
![Screenshot_20230117-121731_Aniyomi](https://user-images.githubusercontent.com/117731277/212858637-141fca52-557c-486b-b5bf-88718378a93e.png)
![Screenshot_20230117-121715_Aniyomi](https://user-images.githubusercontent.com/117731277/212858666-09f76b3c-876d-4f3a-8029-4f71fa9650f3.png)

**Comment by hvroun1** [comment]
@sinnghabhay80 yes, also on my end while streaming it works perfectly just as yours but try completing any download and all subtitles options disappear.

**Comment by FrenchPrick** [comment]
Same thing here superstream only shows subtitles while streaming.
